### PR TITLE
[#151463254] - Add icon to note in UI - so that it would show modific…

### PR DESCRIPTION
**Pull request by mezerny**
Changes for tasks:
[#151463034] - EventLog record must have old value in it
[#151463254] - Add icon to note in UI - so that it would show modifications of the note text

### CODE REVIEW

File: src/partials/notes.html [hunk 1]
```diff
@@ -27,6 +27,9 @@
                     <span class="edit" ng-click="editNote(note, $index)" ng-if="!note.trackable_id" ng-hide="auth.id != note.user_id">
                         <span class="dlicons-pencil" title="Edit note"></span>
                     </span>
+                    <span class="edit" ng-click="showHistory(note)" ng-if="note.versions">
```
Comment (anton-kaidalov) [inline src/partials/notes.html]: In the ticket, Alexei asks to show this icon only if there is some history. You're showing it always.

File: src/partials/note_versions.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                <thead>
+                <tr>
+                    <th class="cell-datetime">Date</th>
+                    <th class="cell-patient">Message</th>
+                    <th class="cell-from">Author</th>
+
+                </tr>
+                </thead>
+                <tbody>
+                    <tr ng-repeat="version in versions" ng-if="version.event === 'update'">
```
Comment (anton-kaidalov) [inline src/partials/note_versions.html]: In the ticket, Alexei asks about "up to 10 last versions". You show them all.

File: src/partials/note_versions.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                        <td>{{version.reify.updated_at | date: 'short'}}</td>
+                        <td>{{version.reify.message}}</td>
+                        <td>{{version.whodunnit}}</td>
+                    </tr>
+                </tbody>
+        </table>
+        <p>{{version.reify.message}}</p>
```
Comment (anton-kaidalov) [inline src/partials/note_versions.html]: What's this? I guess, some random leftover.

**Comment by anton-kaidalov** [comment]
In addition to the comments in code.

Nobody can ever see the last modification date of the note in our app, because the Referral page shows the notes creation dates, and our notes history dialog does not have the very last editing.

Is this really OK for the users?

Recommendation: consider showing the last modification date on the Referral page.

**Comment by anton-kaidalov** [comment]
dates

**Comment by mezerny** [comment]
@anton-kaidalov I think in general this is out of scope for current task. Nevertheless I've fixed described identation.

**Comment by anton-kaidalov** [comment]
Reported https://www.pivotaltracker.com/story/show/152899736.

Closing this PR.

**Pull request merged**